### PR TITLE
Bump version of wasmd to 27.rc3 and update antehandler

### PR DESCRIPTION
**Pull request by the-frey**
Need this in so we can test on testnet ahead of final version.

### CODE REVIEW

File: app/app.go [hunk 1]
```diff
@@ -386,11 +386,18 @@ func New(
...
 	// Create Transfer Keepers
 	app.TransferKeeper = ibctransferkeeper.NewKeeper(
-		appCodec, keys[ibctransfertypes.StoreKey], app.GetSubspace(ibctransfertypes.ModuleName),
-		app.IBCKeeper.ChannelKeeper, &app.IBCKeeper.PortKeeper,
-		app.AccountKeeper, app.BankKeeper, scopedTransferKeeper,
+		appCodec,
+		keys[ibctransfertypes.StoreKey],
+		app.GetSubspace(ibctransfertypes.ModuleName),
+		app.IBCKeeper.ChannelKeeper,
+		app.IBCKeeper.ChannelKeeper,
```
Comment (joeabbey) [inline app/app.go:393]: Do we need two?

Comment (joeabbey) [reply app/app.go:393]: Yeah, just saw: https://github.com/cosmos/ibc-go/blob/v3.0.0/modules/apps/transfer/keeper/keeper.go#L35

**Review by joeabbey — APPROVED** [review]
LGTM, (assuming we're not activating ICA)

**Review by dimiandre — APPROVED** [review]
Lgtm!

**Pull request merged**